crates/chainspec/src/spec.rs

```diff
@@ -1573,7 +1556,7 @@ Post-merge hard forks (timestamp based):
             &DEV,
             &[(
                 Head { number: 0, ..Default::default() },
-                ForkId { hash: ForkHash([0x45, 0xb8, 0x36, 0x12]), next: 0 },
+                ForkId { hash: ForkHash([0xd5, 0xbc, 0xa3, 0xa4]), next: 0 },
```

Comment:
update this test output. since it is dev chain, it shouldn't matter that this was buggy before when manually set.

Reply:
reverted, assuming same pattern as with op mainnet genesis hash